wandb/apis/public.py

```diff
@@ -217,6 +233,17 @@ def execute(self, *args, **kwargs):
                 )
             raise
 
+    @property
+    def server_info(self):
+        if self._server_info is None:
+            self._server_info = self.execute(self.INFO_QUERY).get("serverInfo")
+        return self._server_info
+
+    def version_supported(self, min_version):
```

Comment:
This is a really nice helper -- I think I've scattered in a bunch of ad-hoc checks for the other functions here.